src/screens/NationCreateContainer/index.js

```diff
@@ -120,10 +107,41 @@ class NationCreateContainer extends Component {
         }),
       }]);
   };
+
+  render() {
+    return (
+      <NationCreateScreen
+        {...this.props}
+        onCancelNationCreation={this.cancelNationCreation}
+        onResetNationCreation={this.resetNationCreation}
+        onSaveNationDraft={this.saveForm}
+        onDeleteNationDraft={this.deleteForm}
+        onSubmitNation={this.submitForm}
+      />
+    );
+  }
 }
 
-NationCreateContainer.PropTypes = {
-  navigator: PropTypes.object,
+NationCreateContainer.propTypes = {
+  navigator: PropTypes.shape({ dismissModal: {} }),
```

Comment:
This PropTypes declaration is invalid. You need to use PropTypes constant in nested object as well, like 
`PropTypes.shape({ dismissModal: PropTypes.func })`. 
Do you mind to enable flow for that file and remove PropTypes declaration at all?

Reply:
Will enable `flow` on the next commit and solve this.